packaging/tests/tas-cmis/src/test/resources/search-in-folder.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--CMIS Queries: passing the search query as first param and results expected -->
+<testData xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+	<queries>
+		<query value="SELECT cmis:name, cmis:parentId, cmis:path, cmis:allowedChildObjectTypeIds FROM cmis:folder where IN_FOLDER('%s') AND cmis:name = 'subFolder'" expectedResults="1" />
+		<query value="SELECT cmis:name, cmis:objectId, cmis:lastModifiedBy, cmis:creationDate, cmis:contentStreamFileName FROM cmis:document where IN_FOLDER('%s') AND cmis:name = 'fourthFile'" expectedResults="1" />
+		<query value="SELECT cmis:parentId FROM cmis:folder where IN_FOLDER('%s')" expectedResults="3" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s')" expectedResults="5" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s') ORDER BY cmis:name ASC" expectedResults="5" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s') ORDER BY cmis:name DESC" expectedResults="5" />
+		<query value="SELECT * FROM cmis:folder where IN_FOLDER('%s') ORDER BY cmis:lastModificationDate ASC"  expectedResults="3" />
+		<query value="SELECT * FROM cmis:folder where IN_FOLDER('%s') ORDER BY cmis:lastModificationDate DESC"  expectedResults="3" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s') ORDER BY cmis:createdBy DESC" expectedResults="5" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s') AND cmis:name IS NOT NULL" expectedResults="5" />
+		<query value="SELECT * FROM cmis:folder where IN_FOLDER('%s') AND cmis:name IS NOT NULL" expectedResults="3" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s') AND cmis:name LIKE 'fourthFile'" expectedResults="1" />
+		<query value="SELECT * FROM cmis:folder where IN_FOLDER('%s') AND NOT(cmis:name NOT IN ('subFolder'))" expectedResults="1" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s') AND cmis:name IN ('fourthFile', 'fifthFile.txt')" expectedResults="2" />
+		<query value="SELECT * FROM cmis:document where IN_FOLDER('%s') AND cmis:name NOT IN ('fourthFile', 'fifthFile.txt')" expectedResults="3" />
+		<query value="SELECT * FROM cmis:folder where IN_FOLDER('%s') AND cmis:name &lt;&gt; 'subFolder'" expectedResults="2" />
+		<query value="SELECT cmis:secondaryObjectTypeIds FROM cmis:folder where IN_FOLDER('%s') AND cmis:name = 'subFolder'" expectedResults="1" />
```

Comment:
I dislike this style of test. The error messages will be something like "Expected 1 got 2" and historically people have "fixed" this sort of issue by simply changing 1 to 2 in the test.

Reply:
In this [PR](tests/tas-cmis/src/test/resources/cmis-elastic-suite.xml)

Reply:
I'm not sure PR link is correct ?

Reply:
I'm not sure if we're thinking of the same thing, but the data provider comes from this line: https://github.com/Alfresco/alfresco-community-repo/pull/1125/files#diff-b134b8e722005ac2473b2dab6009d070a5ef36c84790bda11b7d3c7ec404d8b0R54
and then the getQueries comes from here: https://github.com/Alfresco/alfresco-tas-utility/blob/master/src/main/java/org/alfresco/utility/data/provider/XMLTestDataProvider.java

Or do you mean the test data needed for the tests which is created in the @BeforeClass here https://github.com/Alfresco/alfresco-community-repo/pull/1125/files#diff-b134b8e722005ac2473b2dab6009d070a5ef36c84790bda11b7d3c7ec404d8b0R21